simulation/sub8_gazebo/nodes/gazebo_controller.py

```diff
@@ -66,6 +71,16 @@ def publish_height(self, msg):
             )
         )
 
+    def publish_blueview_ping(self, msg):
+        self.blueview_ranges_pub.publish(
+            BlueViewPing(
+                header=msg.header,
+                bearings=[(msg.angle_min + i * msg.angle_increment) for i in xrange(len(msg.ranges))],
+                ranges=msg.ranges,
+                intensities=[500 for x in msg.ranges]
```

Comment:
Wait why the ranges? Just so it visualizes better? Are the intensities used anywhere in the ogrid stack?